examples/imagenet/README.md

```diff
@@ -46,7 +46,7 @@ Configuration flag is defined using
 follows:
 
 ```shell
-python main.py --workdir=./imagenet_default --config.num_epochs=100
+python main.py --workdir=./imagenet_default --config.num_epochs=100 --config=configs/default.py
```

Comment:
This should also be added to line 37, no?

Reply:
tweaked!